Makefile

```diff
@@ -81,7 +81,8 @@ yarn-install: yarn-ensure
 
 .PHONY: backend-integration-test
 backend-integration-test:
-	cd backend/internal/test/integration/xds && docker-compose up --build --abort-on-container-exit
+	docker-compose -f backend/internal/test/integration/serverexperimentation/docker-compose.yaml up --build --abort-on-container-exit
+	docker-compose -f backend/internal/test/integration/xds/docker-compose.yaml up --build --abort-on-container-exit
```

Comment:
maybe there is some clever way to run second docker-compose only if the first one succeeds (all tests pass)?

Reply:
fixed